docs/process/writing-release-notes.md

```diff
@@ -29,7 +29,7 @@ We don't include release notes that don't impact the user's experience using Des
 We'll make exceptions for security vulnerability fixes, especially if they're high profile:
```

Comment:
How do we feel about being explicit and saying "While we won't mention the CVE, we will mention where the security vulnerability was" -- too explicit? It's come up twice so I wondered if it'd be helpful to memorialize